ansible/roles-infra/infra-images/defaults/main.yaml

```diff
@@ -11,6 +11,20 @@ _infra_images_arch: "{{ ocp4_architecture_cluster | default(infra_images_arch) |
 
 infra_images_predefined:
 
+  RHEL-10.0-GOLD-latest:
+    owner: "{{ infra_images_redhat_owner_id }}"
+    name: RHEL-10.0.*_HVM_*Access*
+    architecture: "{{ _infra_images_arch }}"
+    aws_filters:
+      is-public: false
```

Comment:
Can we add also `RHEL-10*-latest` too?

```yaml
  RHEL-10.0-latest:
    owner: "{{ infra_images_redhat_owner_id }}"
    name: RHEL-10.0.*_HVM_*
    architecture: "{{ _infra_images_arch }}"
    aws_filters:
      is-public: true
      
  RHEL-10-latest:
    owner: "{{ infra_images_redhat_owner_id }}"
    name: RHEL-10*_HVM_*
    architecture: "{{ _infra_images_arch }}"
    aws_filters:
      is-public: true

Reply:
Patrick suggested that we don't want to continue supporting that.

Reply:
Added, but left out `RHEL-10.0-latest` as we didn't have equivalent for RHEL 9.